scania-digital-design-system/sdds
# fix(toast): corrected icon type for error state

**Describe pull-request**  
Corrected icon type for error state

**Solving issue**  
Fixes: AB#2662

**How to test**  
1. Go to storybook link below
2. Check in Components -> Toast
3. Check that the error state is using the correct icon.

**Screenshots**  
-

**Additional context**  
-
